### PR TITLE
docstring rendering

### PR DESCRIPTION
**Pull request by Deagler**
## Motivation

(Write your motivation for proposed changes here.)

### Have you read the [Contributing Guidelines on pull requests](https://github.com/move-language/move/blob/main/CONTRIBUTING.md#developer-workflow)?

(Write your answer here.)

## Test Plan

(Share your test plan here. If you changed code, please provide us with clear instructions for verifying that your changes work.)

### CODE REVIEW

File: language/move-analyzer/src/symbols.rs [hunk 1]
```diff
@@ -142,6 +142,8 @@ pub struct UseDef {
     def_loc: DefLoc,
     /// Location of the type definition
     type_def_loc: Option<DefLoc>,
+    /// Doc string for the relevant identifier/function
+    doc_string: String,
```
Comment (0xYao) [inline language/move-analyzer/src/symbols.rs:146]:
```suggestion
    doc_string: Option<String>,
```

File: language/move-analyzer/src/symbols.rs [hunk 2]
```diff
@@ -1626,6 +1671,7 @@ impl Symbolicator {
                                     &use_name,
                                     ident_type,
                                     ident_type_def,
+                                    String::new(),
```
Comment (0xYao) [inline language/move-analyzer/src/symbols.rs]: Shouldn't need to initialize the strings if `doc_string` is of type `Option<String>`.

```suggestion

```

**Review by 0xYao — CHANGES_REQUESTED** [review]
The doc_string is only shown for the methods inside the same file.

**Comment by Deagler** [comment]
> The doc_string is only shown for the methods inside the same file.

Fixed and removed references to String::new

**Pull request closed without merging**